sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeSentimentAsyncClient.java

```diff
@@ -184,7 +183,13 @@ private AnalyzeSentimentResult convertToAnalyzeSentimentResult(DocumentSentiment
                     confidenceScorePerLabel.getNegative(),
                     confidenceScorePerLabel.getNeutral(),
                     confidenceScorePerLabel.getPositive()),
-                new IterableStream<>(sentenceSentiments)));
+                new IterableStream<>(sentenceSentiments),
```

Comment:
Better to have a mapping helper method as it continues to grow. Make the top API impl as simple as possible.

Reply:
Can you give an example?

FYI, This class is not public to user.